packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -71,15 +71,15 @@ class CollapsibleMenuViewItemGroup extends React.Component {
 
     if (isCollapsibleMenuItem && selectedKeys.length) {
       return (
-        <li>
+        <li role="none">
```

Comment:
Pulled this idea from nested menus in this example: https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html